photon-client/src/views/CamerasView.vue

```diff
@@ -653,10 +661,9 @@ export default {
                 const calData = this.calibrationData;
                 calData.isCalibrating = true;
                 data['startPnpCalibration'] = calData;
-
                 console.log("starting calibration with index " + calData.videoModeIndex);
             }
-
+            this.$store.commit('currentPipelineIndex', -2);
```

Comment:
What's odd is that I woulda thought that when the backend transitions into calibration, it'd send a new currentPipelineIndex... fine as is, just curious

Reply:
That's what I thought too. 
Below is a gif of the behavior before and after this commit. Notice how before it requires a refresh of the page to set the currentPipelineIndex to -2. 
![camera_calibration](https://user-images.githubusercontent.com/31329139/198772100-79f84249-1aae-4de8-8dd9-a582e1b5e8d5.gif)

Reply:
Seems like the backend needs to rebroadcast a full settings struct to UI clients when entering calibration mode? For now though this is a fine solution